data/modules_i.json

```diff
@@ -24,32 +26,36 @@
     "CAB": "Majormodul",
     "CAE": "Zusatzmodul",
     "CCNA": "Erweiterungsmodul",
-    "CHMA": "Zusatzmodul",
+    "CHMA": "Erweiterungsmodul",
     "CI": "Majormodul",
     "CILAB": "Majormodul",
     "CNA": "Kernmodul",
     "CNF": "Erweiterungsmodul",
     "CPE_PREP": "Zusatzmodul",
+    "CRS": "Erweiterungsmodul",
     "CSA": "Erweiterungsmodul",
     "CYBER": "Erweiterungsmodul",
     "DASB": "Majormodul",
     "DAVI": "Majormodul",
     "DAWA": "Majormodul",
     "DB&S": "Kernmodul",
     "DBS": "Kernmodul",
+    "DCEX": "Erweiterungsmodul",
     "DEFRC1.2": "Zusatzmodul",
     "DEFRC1_1": "Zusatzmodul",
     "DESG": "Zusatzmodul",
+    "DIBM": "Erweiterungsmodul",
     "DIGITRANS": "Zusatzmodul",
+    "DITI": "Erweiterungsmodul",
     "DL4G": "Majormodul",
     "DMATH": "Kernmodul",
-    "EAEK": "Erweiterungsmodul",
-    "EAUS": "Erweiterungsmodul",
```

Comment:
have those modules been removed from the i modules website?

Reply:
yes, can't find them anymore.